packages/playground/website/project.json

```diff
@@ -19,7 +19,8 @@
 					"cp -r ./client ./wasm-wordpress-net/",
 					"cp -r ./remote/* ./wasm-wordpress-net/",
 					"cp -r ./website/* ./wasm-wordpress-net/",
-					"cat ./remote/.htaccess > ./wasm-wordpress-net/.htaccess"
+					"cp ../../../packages/playground/website/.htaccess ./website/",
```

Comment:
For posterity, we may need to reuse the same vite plugin as `playground/remote` in `playground/website` before merging this one:

```
	/**
	 * Copy the `.htaccess` file to the `dist` directory.
	 */
	{
		name: 'htaccess-plugin',
		apply: 'build',
		writeBundle({ dir: outputDir }) {
			const htaccessPath = path('.htaccess');

			if (existsSync(htaccessPath) && outputDir) {
				copyFileSync(htaccessPath, join(outputDir, '.htaccess'));
			}
		},
	} as Plugin,
```